static/app/views/settings/project/projectToolbar.tsx

```diff
@@ -52,6 +59,18 @@ function ProjectToolbarSettings({organization, project, params: {projectId}}: Pr
         renderDisabled={NoAccess}
       >
         <PermissionAlert project={project} />
+        <Alert type="info" showIcon>
```

Comment:
If there's no domain then no need for the alert at all. 

For example, if someone got here from clicking through project settings.